.babelrc.js

```diff
@@ -50,6 +50,7 @@ module.exports = {
     ],
     ['@babel/plugin-proposal-class-properties', { loose: true }],
     ['@babel/plugin-proposal-private-methods', { loose: true }],
+    ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
```

Comment:
SO this is an interesting one:

I was tweaking around with the output from the compile-babel command, and when it piped stdout to the main process, it started complaining about `loose` not being configured correctly.

It suggested I added this, and when I did the error disappeared. 🎈

Reply:
This was also an issue I faced when using the babel config for the prebundler work.